rust-toolchain.toml

```diff
@@ -1,9 +1,15 @@
 [toolchain]
 channel = "nightly-2024-06-13"
-targets = [
-    "x86_64-unknown-linux-gnu",
-    "i686-unknown-linux-gnu",
-    "aarch64-unknown-linux-gnu",
-    "armv7-unknown-linux-gnueabihf",
-]
+
+# Commented out targets to avoid downloading or caching all these during CI
+# runs. See discussion on rav1d PR #1174. Uncomment to cross-compile locally.
+# targets = [
+#     "aarch64-unknown-linux-gnu",
+#     "aarch64-apple-darwin",
+#     "aarch64-linux-android",
+#     "armv7-unknown-linux-gnueabihf",
+#     "i686-unknown-linux-gnu",
+#     "riscv64gc-unknown-linux-gnu",
```

Comment:
Do we support riscv32, too?

Reply:
I haven't tested it. Took a quick look, short answer is no, dav1d doesn't seem to support riscv32 either. Notes:

The `riscv64gc-unknown-linux-gnu` target is tier 2 whereas `riscv32gc-unknown-linux-gnu` is tier 3 and the nightly we're on doesn't have a build for that target. I also wasn't able to find ubuntu packages with a suitable C compiler and linker for riscv32.